ddtrace/internal/processor/trace.py

```diff
@@ -215,3 +216,33 @@ def shutdown(self, timeout):
         except ServiceStatusError:
             # It's possible the writer never got started in the first place :(
             pass
+
+
+@attr.s
+class SingleSpanSamplingProcessor(SpanProcessor):
+    """SpanProcessor for sampling single spans:
+
+    - This processor must come after trace sampling priority as been applied, as it only assesess spans
+      that will not be kept based of of trace sampling priority.
+    - Users can specify span rules with service, name, sample rate, and rate limit.
+    - Those rules are used to assess all spans that are not sampled by trace sampling.
```

Comment:
My understanding is that this is not as specified but we still need to sort that out.

```suggestion
```